src/components/badge/README.md

```diff
@@ -1 +1,44 @@
-# Badge
\ No newline at end of file
+# Badge
+
+Use badges to call out or label important information in your UI.
+
+>
+    Concise labels
+
+    Because badges are intended to call attention to information, it is important to keep badge text concise. It should not span multiple lines.
```

Comment:
I think this could be added as a section at the bottom: `## Implementation notes`.